CONTRIBUTING.md

```diff
@@ -317,6 +317,24 @@ cd packages/desktop-gui
 npm rebuild node-sass
 ```
 
+#### Docker for built binary
```

Comment:
@bahmutov Don't we already have a docker script that does this?

It's just `npm run docker`. Am I misunderstanding?